authors/apps/comments/renderers.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+from rest_framework.renderers import JSONRenderer
+
+
+class LikeComementsJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+
+        if 'errors' in data or 'detail' in data:
+            return super().render(data)
+
+        return json.dumps({
+            'likes': data,
+            # "likes count": data
```

Comment:
Hello, @Opio-Emmanuel-Omona  I have gone through your PR and realized you left some commented code. I humbly request you remove the comment.
Thank you.

Reply:
@moseskamira I have replaced the commented out code with, a statement that counts the number of likes.